index.html

```diff
@@ -876,41 +876,96 @@ <h3>Directory Service API</h3>
                     incremental transfer of the payloads.
                 </p>
 
+                <p>
+                    The directory APIs include mandatory, recommended, and optional
+                    features. 
+                    Directories that don't offer recommended or optional features
+                    inform the clients about the absence of those features
+                    according to the following rules:
+                    <ul>
+                        <li>
+                            If the missing feature is to customize existing functionality of an
+                            API, the server will respond with 501 (Not Implemented)
```

Comment:
will -> MUST assertion?

Reply:
Assertions are in the respective sections as explained below:
> The normative behavior is prescribed in the respective sections.

Reply:
Assertions are in the respective sections as explained below:
> The normative behavior is prescribed in the respective sections.